quinn-proto/src/packet.rs

```diff
@@ -323,6 +323,12 @@ impl ::std::ops::DerefMut for ConnectionId {
     }
 }
 
+impl fmt::Debug for ConnectionId {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
```

Comment:
What do you think about using a more compact representation, like I used to have?

https://github.com/djc/quinn/blob/pre-quicr-quinn/src/types.rs#L35

Also, since it's very easy to do, maybe add a unit test for this?

Reply:
That looks a lot like the existing `Display` impl. I believe it's idiomatic for `Debug` impls to produce valid rust syntax, for e.g. easy copy-pasting into a unit test.

Reply:
Ah, yeah, you're right.

Though in that case I'd be in favor of using `Display` rather than `Debug` in log messages (but that can be a different change -- maybe a first good patch?).

Reply:
Isn't that mostly already being done?

Reply:
Ah, I suppose you're right. I guess I got confused about the logging in #76.